.github/workflows/post-merge-pipeline.yml

```diff
@@ -0,0 +1,156 @@
+      - name: Archive the build artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: web-components-artifact
+          path: ./${{ env.WEB_COMPONENTS_DIR }}/web-components.tar.gz
+  docker-build-web-components:
+    runs-on: [infra1-small]
+    needs: [build-web-components]
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Set SHORT SHA and BRANCH ESCAPE REF NAME to ENV Variable to use for image tag later
+        run: |
+          echo "GITHUB_SHORT_SHA=${GITHUB_SHA::8}" >> $GITHUB_ENV
+          echo "GITHUB_ESCAPE_REF_NAME=${GITHUB_REF_NAME/\//-}" >> $GITHUB_ENV
+      - name: Set up QEMU to add multiplatform support
+        uses: docker/setup-qemu-action@v2
+      - name: Create docker context
+        run: |
+          docker context create builder
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          endpoint: builder
+      - name: Get Vault secrets for ECR
+        uses: hashicorp/vault-action@v2
+        with:
+          url: "https://vault.ord1.infra.intgdc.com"
+          method: jwt
+          path: jwt/github
+          role: ecr-push
+          secrets: |
+            secret/data/v2/data-special/infra1-user-ecr-rw aws_ecr_access_key | AWS_ACCESS_KEY ;
+            secret/data/v2/data-special/infra1-user-ecr-rw aws_ecr_secret_key | AWS_SECRET_KEY ;
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-access-key-id: ${{ env.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ env.AWS_SECRET_KEY }}
+          aws-region: us-east-1
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+      - uses: actions/download-artifact@v3
+        with:
+          name: web-components-artifact
+          path: ./${{ env.WEB_COMPONENTS_DIR }}
+      - name: Docker build and push image
+        uses: docker/build-push-action@v4
+        with:
+          context: ./${{ env.WEB_COMPONENTS_DIR }}
+          file: ./${{ env.WEB_COMPONENTS_DIR }}/Dockerfile
+          platforms: |
+            linux/amd64
+            linux/arm64
+          push: true
+          tags: |
+            020413372491.dkr.ecr.us-east-1.amazonaws.com/nas-testing/${{ github.event.repository.name }}:${{ env.GITHUB_ESCAPE_REF_NAME }}
+            020413372491.dkr.ecr.us-east-1.amazonaws.com/nas-testing/${{ github.event.repository.name }}:${{ env.GITHUB_SHORT_SHA }}
+    outputs:
+      github_short_sha: ${{ env.GITHUB_SHORT_SHA }}
+      docker_username: ${{ steps.login-ecr.outputs.docker_username_020413372491_dkr_ecr_us_east_1_amazonaws_com }}
+      docker_password: ${{ steps.login-ecr.outputs.docker_password_020413372491_dkr_ecr_us_east_1_amazonaws_com }}
+#  generate-update:
```

Comment:
commented lines

Reply:
This commented lines will be uncommented in another PR after this PR merged, if you are ok to keep it like this.
The purpose of this job is to create the merge request to gitlab gdc-nas repos for updating version txt file of web-components

Reply:
Ok then :)